src/vue/modules/movements-history/components/movements-table-row.vue

```diff
@@ -1,6 +1,22 @@
 <template>
   <tbody class="movements-table-row">
-    <tr class="movements-table-row__brief">
+    <tr
+      v-if="isEmptyMovements"
+      class="movements-table-row__brief
+             movements-table-row__no-data"
+    >
```

Comment:
the no-data of the movements should be extracted into a new component, cuz now every time you try to render the row, you check for `isEmptyMovements`

pls do not forget to restore the initial look of the current file when you extract the no-data part